packages/web-runtime/src/pages/account.vue

```diff
@@ -335,11 +336,16 @@ export default defineComponent({
         const languageConfiguration = unref(valuesList)?.find(
           (cV) => cV.identifier.setting === 'language'
         )
-        selectedLanguageValue.value = languageConfiguration
-          ? unref(languageOptions)?.find(
-              (lO) => lO.value === languageConfiguration.value?.listValue?.values?.[0]?.stringValue
-            )
-          : unref(languageOptions)?.find((o) => o.default)
+
+        if (languageConfiguration) {
+          selectedLanguageValue.value = unref(languageOptions)?.find(
+            (lO) => lO.value === languageConfiguration.value?.listValue?.values?.[0]?.stringValue
+          )
+        } else {
+          selectedLanguageValue.value = unref(languageOptions).find(
+            (lO) => lO.value === navigator.language
```

Comment:
This is what I meant. Please no `navigator.language` - that's already handled when the translations are being initialized.

```suggestion
            (lO) => lO.value === langauge.defaultLanguage
```

Reply:
yes, saw that, but I even dont know if we take this solution at all, this ha also some downsides

Reply:
not language.defaultLanguage but language.current

Reply:
I don't think thats the way to go, I guess current will fallback to en if its a language we don't support, so we will have set English there istead of Please choose, idk

Reply:
ok checked with zh, current is fine